infrastructure/ai/providers/openai_provider.py

```
@@ -46,6 +47,17 @@ def __init__(self, settings: Settings):
 
         self.async_client = AsyncOpenAI(**client_kwargs)
 
+    def _persist_legacy_flag(self, use_legacy: bool) -> None:
+        """将 use_legacy_chat_completions 标志持久化到数据库（仅当有 profile_id 时）。"""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)